library/database/class.sqldriver.php

```diff
@@ -2141,6 +2141,9 @@ public function _whereIn($field, $values, $op = 'in', $escape = true) {
         if (count($in) > 0) {
             $inExpr = '('.implode(', ', $in).')';
         } else {
+            if ($op == 'not in') {
+                deprecated('Gdn_SQLDriver::whereNotIn() with empty $values array');
```

Comment:
```suggestion
                deprecated('Gdn_SQLDriver::whereNotIn() was called with empty $values array. This will no longer be supported in a future release.');
                \Vanilla\Utility\Deprecation::log();
```